tests/test_capacity_analysis.py

```diff
@@ -91,64 +91,88 @@ def setUpClass(cls) -> None:
 
     def test_capacity_analysis_stats(self) -> None:
         self.assertEqual(0, len(CapacityAnalysisStats.contingencies_dict()))
-        self.assertEqual(21, len(CapacityAnalysisStats.feasibility_dict()))
-        self.assertEqual(
-            [
-                UnfeasibleCondition(
-                    -80 - 38.74576838702821j,
-                    LimitingFactor(
-                        Violations.BUS_UNDERVOLTAGE | Violations.BUS_OVERVOLTAGE,
-                        ss=None,
+        if sys.platform == "win32" and not envs.pandapower_backend:
+            self.assertEqual(21, len(CapacityAnalysisStats.feasibility_dict()))
+            self.assertEqual(
+                [
+                    UnfeasibleCondition(
+                        -80 - 38.74576838702821j,
+                        LimitingFactor(
+                            Violations.BUS_UNDERVOLTAGE | Violations.BUS_OVERVOLTAGE,
+                            ss=None,
+                        ),
                     ),
-                ),
-                UnfeasibleCondition(
-                    -15 - 7.264831572567789j,
-                    LimitingFactor(
-                        Violations.BUS_UNDERVOLTAGE | Violations.BUS_OVERVOLTAGE,
-                        ss=None,
+                    UnfeasibleCondition(
+                        -15 - 7.264831572567789j,
+                        LimitingFactor(
+                            Violations.BUS_UNDERVOLTAGE | Violations.BUS_OVERVOLTAGE,
+                            ss=None,
+                        ),
                     ),
-                ),
-            ],
-            CapacityAnalysisStats.feasibility_dict()[
-                Bus(number=101, ex_name="NUC-A       21.600", type=2)
-            ][0:-1:3],
-        )
+                ],
+                CapacityAnalysisStats.feasibility_dict()[
+                    Bus(number=101, ex_name="NUC-A       21.600", type=2)
+                ][0:-1:3],
+            )
+        else:
+            self.assertEqual(19, len(CapacityAnalysisStats.feasibility_dict()))
+            self.assertEqual(
+                [],
+                CapacityAnalysisStats.feasibility_dict()[
+                    Bus(number=101, ex_name="NUC-A       21.600", type=2)
+                ][0:-1:3],
```

Comment:
The PandaPower has a different numbering. The PSSE bus number is 101 is a PandaPower bus number 100. The other fields are slightly different to.
```suggestion
                CapacityAnalysisStats.feasibility_dict()[
                    Bus(number=100, ex_name="21.6 77.0", type=1)
                ],
```
And there are less unfeasible conditions. So we don't need the `[0:-1:3]` indexing.